tests/10-deploy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+import amulet
+import requests
+import unittest
+
+
+class TestCharm(unittest.TestCase):
+    def setUp(self):
+        self.d = amulet.Deployment()
+
+        self.d.add('bundleservice-charm')
+        self.d.expose('bundleservice-charm')
+
+        self.d.setup(timeout=900)
+        self.d.sentry.wait()
+
+        self.unit = self.d.sentry['bundleservice-charm'][0]
+
+    def test_service(self):
+        # test we can access over http
+        page = requests.get('http://{}'.format(self.unit.info['public-address']))
```

Comment:
Is port 8000 part of 'public-address'?  Where does it come from?  It would be a better test to see it responding on port 8000 since that is what is hard-coded in the charm.